src/logic/map_objects/descriptions.h

```diff
@@ -17,71 +17,97 @@
  *
  */
 
-#ifndef WL_LOGIC_MAP_OBJECTS_TRIBES_TRIBES_H
-#define WL_LOGIC_MAP_OBJECTS_TRIBES_TRIBES_H
+#ifndef WL_LOGIC_MAP_OBJECTS_DESCRIPTIONS_H
+#define WL_LOGIC_MAP_OBJECTS_DESCRIPTIONS_H
 
 #include <list>
 #include <memory>
 
 #include "base/macros.h"
+#include "io/filesystem/filesystem.h"
 #include "logic/map_objects/description_maintainer.h"
 #include "logic/map_objects/description_manager.h"
-#include "logic/map_objects/immovable.h"
 #include "logic/map_objects/map_object_type.h"
-#include "logic/map_objects/tribes/building.h"
-#include "logic/map_objects/tribes/ship.h"
-#include "logic/map_objects/tribes/tribe_descr.h"
-#include "logic/map_objects/tribes/ware_descr.h"
-#include "logic/map_objects/tribes/worker_descr.h"
+#include "logic/map_objects/tribes/wareworker.h"
+#include "map_io/tribes_legacy_lookup_table.h"
 #include "scripting/lua_table.h"
 
 namespace Widelands {
 
-class Tribes {
+class BuildingDescr;
+class CritterDescr;
+class ImmovableDescr;
+class ResourceDescription;
+class ShipDescr;
+class TerrainDescription;
+class TribeDescr;
+class WareDescr;
+class WorkerDescr;
+
+class Descriptions {
```

Comment:
Another small nit: For a class that is used a lot this is a pretty awkward name to type imho. How about renaming it to something easier to type, e.g. `Units`? (only if you agree)

Reply:
I was thinking the same, because that's the name of the Tribes' main Lua files now, but I then I decided to keep the naming consistent with `ShipDescr` etc, to signal that this class contains `ShipDescr` etc, not` Ship` etc. We also have `DescriptionIndex` and `DescriptionMaintainer`.